functions/mqhandlers.go

```diff
@@ -164,8 +164,17 @@ func HostPeerUpdate(client mqtt.Client, msg mqtt.Message) {
 		gwDelta,
 		&originalGW,
 	)
-
-	go handleEndpointDetection(&peerUpdate)
+	// handle endpoint detection
+	if peerUpdate.Host.EndpointDetection != config.Netclient().Host.EndpointDetection {
```

Comment:
I believe we decided that it's best that the `EndpointDetection` config should be moved to the server config

Reply:
Wrong merge, fixed.